backend/Tasque.BLL/Services/EntityCrudService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Tasque.Core.Common.Entities.Abstract;
+using Tasque.Core.DAL;
+
+namespace Tasque.Core.BLL.Services
+{
+    public abstract class EntityCrudService<TEntity> : IEntityCrudService<TEntity> where TEntity : BaseEntity
```

Comment:
Adding service in both #20 and #21?

Reply:
I decided to split creations of base Service and Controller at different tasks, hence, different branches. However, branch 46 is based on branch 45, so it seems like it copies the content of 45.

Reply:
I based 46 on 45 instead of plain dev to prevent blocking of the working process by waiting time.

Reply:
So maybe close #20 and proceed with this one?

Reply:
Makes sence. I will reopen it if needed.